blog/designing-for-journey-candance-van-oostrum.md

```diff
@@ -0,0 +1,35 @@
+
+This personal transformation took time. There were bumps, and I learned some really valuable lessons the hard way. The most important lesson I learned was how to build meaningful relationships across the aisle. I can say with a high degree of confidence that I owe a great part of my success to collaborating, pairing and partnering with engineers.  
+
+What I love most about my job is that product designers cut across entire organizations. The success of the role is largely dependent on building trust and establishing true partnerships with key stakeholders. Design interfaces with growth marketing, dev rel, product, engineering and customer success. This gives us a really unique vantage point inside of a business, as we synthesize pain points, needs and business challenges across multiple business units. Contrast this with the visibility of prod-eng teams, who largely operate within their own scope, mandate and service area ownership. While these prod-eng teams have a very deep and specific understanding of their services and use cases, they have a limited knowledge of how other services or parts of the product work.
+
+As companies grow, the service topology, dependencies and complexities grow hand-in-hand. Along with this growth, comes increased debt (both ux and tech), high on-call volumes, constant interruptions, unplanned work, and weak or unreliable institutional knowledge. Not only do these issues surface in the UX, but it’s often put on the product design teams to fix them, without fixing the underlying problems.
+
+For example, I was once working with a prod-eng team to improve the usability of a core feature and we found, to our surprise, that when the user finished a 3 step create flow, the object wasn’t immediately created. The user was redirected back to the landing page without receiving a success message or any reference of the newly created item, because it in fact had not yet been created due to our adoption of eventual consistency. Unbeknown to the user, the request went into a queue and would eventually be created, and the user would proceed to re-try to create the item. .
+
+What happens next depends on how long it takes for the queue to process the request. In most cases, by the time the user tried to re-enter the name input, the inline validation would throw an error that the name already exists. We observed that users would slightly change the name to satisfy the error message and continue, now adding a 2nd item into the queue and re-entering this loop. In other scenarios, users would move to the final step in the process, hit the create button and would get some gnarly errors because the name already exists and wasn’t caught on the first step, because it was still in the queue.
```

Comment:
```suggestion
What happens next depends on how long it takes for the queue to process the request. In most cases, by the time the user tried to re-enter the name input, the inline validation would throw an error that the name already exists. We observed that users would slightly change the name to satisfy the error message and continue, now adding a second item to the queue and re-entering this loop. In other scenarios, users would move to the final step in the process, hit the create button, and get some gnarly errors because the name already existed and wasn’t caught in the first step because it was still in the queue.
```